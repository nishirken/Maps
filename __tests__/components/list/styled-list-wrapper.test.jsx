import StyledListWrapper from 'Components/list/styled-list-wrapper';
import 'jest-styled-components';

describe('Styled list wrapper', () => {
    const TestStyledListWrapper = shallow(<StyledListWrapper />);

    it('Rendered and change opacity and transform styles onMouseOver', () => {
        TestStyledListWrapper.setProps({ mouseEnter: false });
        expect(TestStyledListWrapper).toMatchStyledComponentsSnapshot();
        TestStyledListWrapper.setProps({ mouseEnter: true });
        expect(TestStyledListWrapper).toMatchStyledComponentsSnapshot();
    });
});
