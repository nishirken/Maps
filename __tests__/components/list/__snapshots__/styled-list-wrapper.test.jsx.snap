// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styled list wrapper Rendered and change opacity and transform styles onMouseOver 1`] = `
.jpjyla {
  position: absolute;
  z-index: 2;
  top: 10%;
  right: 5%;
  overflow-x: visible;
  width: 12.5em;
  height: 80%;
  opacity: 0.5;
  transform: scale(0.95);
  -ms-transform: scale(0.95);
  -webkit-transform: scale(0.95);
  transition: all 0.4s ease-in-out;
  -webkit-transition: all 0.4s ease-in-out;
}

<div
  className="jpjyla"
/>
`;

exports[`Styled list wrapper Rendered and change opacity and transform styles onMouseOver 2`] = `
.kYsViv {
  position: absolute;
  z-index: 2;
  top: 10%;
  right: 5%;
  overflow-x: visible;
  width: 12.5em;
  height: 80%;
  opacity: 1;
  transform: scale(1);
  -ms-transform: scale(1);
  -webkit-transform: scale(1);
  transition: all 0.4s ease-in-out;
  -webkit-transition: all 0.4s ease-in-out;
}

<div
  className="kYsViv"
/>
`;
